Modules/diagnostics/Diagnostics.py

```diff
@@ -0,0 +1,71 @@
+"""
+Diagnostics.py - Diagnostics module
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+
+"""
+
+import logging
+from datetime import datetime
+
+import IPython
+
+import config
+from Modules import permissions
+from Modules.commandcontext import CommandContext
+from Modules.permissions import require_permission
+from Modules.rat_command import Commands
+
+log = logging.getLogger(f'{config.Logging.base_logger}.handlers')
+
+
+@require_permission(permissions.TECHRAT)
+@Commands.command("debug-fetch")
+async def cmd_testNick(bot, trigger: CommandContext):
```

Comment:
I believe we were chatting once about keeping the actual commands separate from the infrastructure.
In that spirit, perhaps a better place for this file would be in `/commands/debug.py`?